Verbose Readme.md

tongrhj opened the pull request:
Readme has been updated to include contributors, tools and frameworks, etc.

tongrhj committed:
Update README.md

ghost commented:
Suggest a tweak to **target audience** and **our value-add** to them:
instead of "hesitant Singaporeans to pick up coding",
consider replacing with "**programmers** to **hone their skills**"

Full revised sentence would read:
"A meteor web app to empower programmers to hone their skills by providing them with a comprehensive directory of programming Tutors (Code Shifus) that they can consult."

Open to further discussion!  For example if Code Shifu is meant not just to **hone** users' skills but also for them to **gain** new ones, we could tweak the sentence to reflect that as well.

Rest of the readme - LGTM :+1:!

tongrhj committed:
Updated description per @jerrygraced's suggestions

Now reads: A meteor web app to empower programmers to hone their skills by providing them with a comprehensive directory of programming tutors (Code Shifus) that they can consult.

tongrhj committed:
Fixed formatting of 'Comprises' section
